iothub_client/inc/iothub_device_client_ll.h

```diff
@@ -29,16 +29,19 @@
 #include "azure_macro_utils/macro_utils.h"
 #include "umock_c/umock_c_prod.h"
 
-#include "iothub_transport_ll.h"
 #include "iothub_client_core_ll.h"
+#include "iothub_transport_ll.h"
+#include "iothub_twin.h"
 
 #ifdef __cplusplus
 extern "C"
 {
 #endif
 
+#ifndef IOTHUB_DEVICE_CLIENT_LL_INSTANCE_TYPE
```

Comment:
humm how does this happen because of line 23?

Reply:
This approach is elsewhere, but I'll remove to keep my changes specific to if-not-version changes.